src/statman_app.erl

```diff
@@ -17,6 +17,13 @@ start(_StartType, _StartArgs) ->
         false ->
             ok
     end,
+    case application:get_env(statman, start_aggregator, true) of
+        true ->
+            statman_aggregator_sup:add_worker(statman_aggregator),
+            statman_server:add_subscriber(statman_aggregator);
```

Comment:
Shall we do a pattern matching here as well to ensure it is started when requested?
```
{ok, _} = statman_aggregator_sup:add_worker(statman_aggregator),
ok = statman_server:add_subscriber(statman_aggregator);
```

Reply:
Sure